app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -9,7 +9,7 @@ def create_in_provider(manager_id, params)
 
       refresh(manager)
       find_by!(:manager_id => manager.id, :manager_ref => tower_object.id)
-    rescue AnsibleTowerClient::ClientError, ActiveRecord::RecordNotFound => error
+    rescue Exception => error
```

Comment:
In general, we don't want to catch `Exception`. But since 
* We always re-raise it 
* If we don't, we'll be sending successful notification
So will this still be ok? I am open to suggestions.